openapi.yaml

```diff
@@ -1501,6 +1523,12 @@ components:
           title: Current page number
           type: string
           example: "1"
+        editable:
```

Comment:
Does `editable` and `deletable` exist on all requests? If yes, should they also be added to `aclRequest`?

Reply:
![tenor-46718972](https://user-images.githubusercontent.com/943800/220168500-74d3841d-e4c5-44d5-9bc2-e5d9e41f5d0b.gif)

Yes, it does. I did add it in some iteration... but seems like I did too much reset and stashing etc. to at least keep the commit for `openapi.yaml` and the auto-generated types in separation. 

Will add it! :D

Reply:
Updated! 
https://github.com/aiven/klaw/pull/654/commits/88f15d769e9c87d9762cbbd2329dccacd6d370a6